Content.Shared/Movement/Systems/SharedSpriteMovementSystem.cs

```diff
@@ -0,0 +1,23 @@
+using Content.Shared.Movement.Components;
+using Content.Shared.Movement.Events;
+
+namespace Content.Shared.Movement.Systems;
+
+public abstract class SharedSpriteMovementSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SpriteMovementComponent, SpriteMoveEvent>(OnSpriteMoveInput);
+    }
+
+    private void OnSpriteMoveInput(Entity<SpriteMovementComponent> ent, ref SpriteMoveEvent args)
+    {
+        if (ent.Comp.IsMoving == args.IsMoving)
+            return;
+
+        ent.Comp.IsMoving = args.IsMoving;
+        Dirty(ent);
```

Comment:
I don't think this even needs to be networked? It's just getting derived from inputmover data.

Reply:
SteeringSystem serveronly

Reply:
Oh, i get it, yes, not needed for input mover

Reply:
@metalgearsloth  If I remove Dirty(), it won't work anymore. Neither when the creature is controlled by the AI, nor when the player